ci/Makefile

```diff
@@ -1,13 +1,11 @@
-.PHONY: run restart-proxy restart-batch-proxy restart-all-proxies
-.PHONY: build-hail-ci push-hail-ci test-locally
+.PHONY: run restart-proxy
+.PHONY: setup-conda-env build-hail-ci push-hail-ci test-locally
```

Comment:
There shouldn't be any conda rules? correct?

Reply:
yep, whoops!